readme.md

```diff
@@ -1,8 +1,63 @@
+qr.dart is a QR code generation library written in Dart.
+
 [![Build Status](https://travis-ci.org/kevmoo/qr.dart.svg?branch=master)](https://travis-ci.org/kevmoo/qr.dart)
 
-## Dart QR code generation library.
+# Features
+
+- Supports QR code versions 1 - 40
+- Error correction / redundancy
+
+# Installing
+
+You can install the package by adding the following lines to your `pubspec.yaml`:
+
+```yaml
+dependencies:
+  qr: "^0.1.0+1"
+```
+
+After adding the dependency to your `pubspec.yaml` you can run: `pub get` or `flutter packages get` if you're using Flutter.
+
+# Getting started
+
+To start, import the dependency in your code:
+
+```dart
+import 'package:qr/qr.dart';
+```
+
+Next, define your QR code data you should do so as such:
+
+```dart
+final _qrCode = new QrCode(4, QrErrorCorrectLevel.L);
+_qrCode.addData("Hello, world in QR form!");
+_qrCode.make();
+```
+
+Now you can use your `_qrCode` instance to render a graphical representation of the QR code. A basic implementation would be as such:
+
+```dart
+for (int x = 0; x < _qrCode.moduleCount; x++) {
+	for (int y = 0; y < _qrCode.moduleCount; y++) {
```

Comment:
use 2 spaces – not tabs...

Reply:
Bah. I thought, for some reason, that github was smart enough to fix that when I saw the tabs. Fixed.